src/js/App.js

```diff
@@ -1,23 +1,78 @@
 import ListView from './ListView.js';
+import FormManager from './FormManager.js';
+import Notification from './Notification.js';
 
 class App {
   constructor() {
-    this.playersScores = [
-      { name: 'some name', score: 100 },
-      { name: 'some name 2', score: 80 },
-      { name: 'some name 3', score: 60 },
-      { name: 'some name 4', score: 40 },
-      { name: 'some name 5', score: 20 },
-      { name: 'some name 6', score: 0 },
-    ];
-
     this.listView = new ListView();
+    this.form = new FormManager();
+    this.submitButton = document.getElementById('submit-button');
+    this.refreshButton = document.getElementById('refresh-button');
+    this.notifications = new Notification();
+
+    this.form.submitData = () => {
+      this.uploadRecord();
+    };
+    this.refreshButton.onclick = () => {
+      this.listView.clearList();
+      this.loadRecords();
+    };
   }
 
   init() {
-    this.playersScores.forEach((player) => {
-      this.listView.addScoreOf(player);
-    });
+    this.loadRecords();
+  }
+
+  loadRecords() {
+    fetch('https://us-central1-js-capstone-backend.cloudfunctions.net/api/games/PonNABo4rAmKIgYZvp3D/scores/')
+      .then((response) => response.json())
+      .then((data) => {
+        data.result.forEach((player) => {
+          this.listView.addScoreOf(player);
+        });
```

Comment:
- [ ] In this milestone, please use `async `await `JavaScript features when making your API calls 🙏